institutions-api/src/main/resources/application.conf

```diff
@@ -20,7 +20,7 @@ hmda {
     createSchema = false
     createSchema = ${?INSTITUTION_PG_CREATE_SCHEMA}
     timeout = 3
-    yearsAvailable = "2018,2019,2020,2021,2022,2023"
+    yearsAvailable = "2018,2019,2020,2021,2022,2023,2024"
     yearsAvailable = ${?INSTITUTION_YEARS_AVAILABLE}
```

Comment:
if we have a configmap or something that sets this variable, make sure to update that as well.